db/db_impl_compaction_flush.cc

```diff
@@ -502,7 +498,7 @@ Status DBImpl::AtomicFlushMemTablesToOutputFiles(
     auto sfm = static_cast<SstFileManagerImpl*>(
         immutable_db_options_.sst_file_manager.get());
     for (int i = 0; i != num_cfs; ++i) {
-      if (cfds[i]->IsDropped()) {
+      if (cfds[i]->IsDropped() || jobs[i].GetMemTables().empty()) {
```

Comment:
ProcessAtomicFlushGroup maybe expand some empty cf, maybe we should avoid these cases in there or early